pkg/config/model.go

```diff
@@ -256,6 +259,8 @@ func DeepCopy(s Spec) Spec {
 	return data
 }
 
+type Status interface{}
```

Comment:
why not use the v1alpha1.Status as the type?

Reply:
Can we assume all CRDs, even ones we do not control, conform to this? I am pretty sure no, since we will need to support this very soon: https://github.com/kubernetes-sigs/service-apis/blob/master/apis/v1alpha1/gateway_types.go#L394.

Reply:
oh okay. Then it has to be generic.

Reply:
cant seem to find that type. Where is it?